config/locales/en.yml

```diff
@@ -212,6 +214,43 @@ en:
       sample:
         one: Sample
         other: Samples
+  activity:
+    member:
+      create_html: "%{user} added member %{href} to %{namespace_type}"
+      destroy_html: "%{user} removed member <span class='text-slate-800 dark:text-slate-300 font-medium'>%{member}</span> from %{namespace_type}"
+      destroy_self_html: "<span class='text-slate-800 dark:text-slate-300 font-medium'>%{member}</span> removed themself from %{namespace_type}"
+      update_html: "%{user} updated %{namespace_type} member %{href} attributes"
+    namespace_group_link:
+      create_html: "%{user} shared %{namespace_type} with group %{href}"
+      destroy_html: "%{user} unshared %{namespace_type} with group <span class='text-slate-800 dark:text-slate-300 font-medium'>%{group_name}</span>"
+      update_html: "%{user} updated share attributes between %{namespace_type} and group %{href}"
+    namespaces_project_namespace:
+      create_html: "%{user} created project"
+      destroy_html: "%{user} removed project"
+      samples:
+        attachment:
+          create_html: "%{user} attached files to sample %{href}"
+          destroy_html: "%{user} removed files from sample %{href}"
+        clone_html: "%{user} cloned samples to project %{href}"
+        cloned_from_html: "%{user} cloned samples from project %{href}"
+        create_html: "%{user} created sample %{href}"
+        destroy_html: "%{user} removed sample <span class='text-slate-800 dark:text-slate-300 font-medium'>%{sample_name}</span>"
+        destroy_multiple_html: "%{user} removed <span class='text-slate-800 dark:text-slate-300 font-medium'>%{deleted_count}</span> samples from project"
+        metadata:
+          update_html: "%{user} updated sample %{href} metadata"
+        transfer_html: "%{user} transferred samples to project %{href}"
+        transferred_from_html: "%{user} transferred samples from project %{href}"
+        update_html: "%{user} updated sample %{href} attributes"
+      transfer_html: "%{user} transferred project from namespace <span class='text-slate-800 dark:text-slate-300 font-medium'>%{old_namespace}</span>"
```

Comment:
**nitpick (blocking):** Can you update this to include the puid of the `new_namespace`, and for `old_namespace` can we also display the `puid` instead of the name.